kitsune/questions/tests/test_models.py

```diff
@@ -364,19 +363,19 @@ def test_from_url(self):
 
         self.assertEqual(q, Question.from_url("/en-US/questions/%s" % q.id))
         self.assertEqual(q, Question.from_url("/es/questions/%s" % q.id))
-        self.assertEqual(q, Question.from_url("/questions/%s" % q.id))
 
     def test_from_url_id_only(self):
-        """Verify question returned from valid URL."""
+        """Verify question returned from URL."""
         # When requesting the id, the existence of the question isn't checked.
         self.assertEqual(123, Question.from_url("/en-US/questions/123", id_only=True))
         self.assertEqual(234, Question.from_url("/es/questions/234", id_only=True))
-        self.assertEqual(345, Question.from_url("/questions/345", id_only=True))
```

Comment:
This is because we use `is_valid_path` in the `Question.from_url` class-method, and for that to return a "truthy" result, a match object, the path has to start with a locale. Also, since we currently use `Question.from_url` only when working with paths we get from Google Analytics, those paths are guaranteed to include the locale.